.github/workflows/docs.yml

```
@@ -36,7 +36,7 @@ jobs:
       - name: Build 1.x docs
         run: |
           cd docs/archive/docs-1.x
-          make html
+          make html	
```

```suggestion
          make html
```
Removed trailing whitespace.